pkg/notify/config/types.go

```diff
@@ -178,102 +242,99 @@ type EmailConfig struct {
 	TLS          *v2alpha1.TLSConfig
 }
 
-func NewEmailReceiver() Receiver {
-	return &Email{
-		common: &common{},
+func NewEmailReceiver(c *Config, er *v2alpha1.EmailReceiver) Receiver {
+	e := &Email{
+		common:   &common{},
+		To:       er.To,
+		Selector: er.AlertSelector,
```

Comment:
All receiver seems to have `Selector`, can we put AlertSelector to common as well?
Maybe we could use a different name instead of `common` such as `ReceiverOption`

Reply:
The `common` is a private struct, The `AlertSelector` will be used in another package, it’s inappropriate to put the `AlertSelector` in the `common` struct.